lib/sidekiq_prometheus.rb

```diff
@@ -118,6 +134,20 @@ def registry
     @registry ||= client::Registry.new
   end
 
+  ##
+  # Register custom metrics
+  # Internal method called by +setup+. This method should not be called from application code in most cases.
+  def register_custom_metrics
+    return if custom_metrics.empty?
+
+    expected_keys = %i[name type docstring].sort
+    if custom_metrics.all? { |m| (m.keys.sort & expected_keys) == expected_keys }
+      SidekiqPrometheus::Metrics.register_metrics(custom_metrics)
+    else
+      Sidekiq.logger.warn('[SidekiqPrometheus] Skipping custom metrics. Hash does not define required keys')
```

Comment:
How about raising an exception instead of a warning? If custom metrics are skipped, the workers will crash later when trying to push data to the inexistent metrics. I'd rather have it crash during setup than later.

Reply:
That makes sense. In that case, I can skip the "validation" and just let Prometheus::Client raise if the metrics hash is malformed.

Reply:
@DuoSRX removed that logic in 714029a